### PR TITLE
Adds comments

### PR DESCRIPTION
**Pull request by carolinux**
Created a pull request to show inline comments, you don't need to merge. 

Some high level comments:

- If at the end (in slicer.py) you use pandas to load the intermediate csvs (so to load them in memory all at once), then it's not clear to me what benefit you get by writing functions to read the initial csvs line by line in reader.py.

- Default arguments are good, but it makes it hard to follow what's happening when the functions are called from main without any explicit arguments. I would take the time to write them out explicitly. That way when you say, ```output = "foo.csv"```, you can reuse this variable when you need to read this output file, and not have to remember to put the same default argument in both the writer and the reader.

- Related to the previous point: It looks like the process has only a few steps. I would not use luigi for that, I would just make sure that the output of the reader.py step is fed into the slicer.py directly

- A public dropbox link is a fine alternative for providing the data for download

- It would help me for our session next week if you share more information about how long this takes to run, for what size of files

### CODE REVIEW

**Comment by carolinux** [comment]
For chaining tasks I would simply do:
```
if expected_output does not exist:
    run_task()
```

And perhaps add a -force option if we want to clean and rerun everything. Let me know if this simple approach suits.

File: reader.py [hunk 1]
```diff
@@ -130,13 +140,20 @@ def get_csv_lines(filename=SOURCE_CSV_PATH, cols=COLNAMES):
 #           
 # 
 
+# COMMENT: I suppose here you are not using pandas because of memory concerns?
+# Because getting the rows as dictionaries is very easy in a pandas DataFrame
```
Comment (epogrebnyak) [inline reader.py]: I had an implementation, where I used pandas to import source data file, put it had several drawbacks of different severety:
- one source csv file has a line that is +column than the rest of the fail, it fails on pandas import 
- memory is concern
- typing is a bit opaque in pandas - I often get float as a result of operation on ints
- dtype is not compatible with skipfooter (C/python parsers)
-  SettingWithCopyWarning: A value is trying to be set on a copy of a slice from a DataFrame is annoying. 
... so as a sum of the above with a dictionary I feel more in control of what is happening. However dequote() and okved3() fucntions can easilly be done in pandas, I think. Each of this functions returns a tuple or a list, how can one use a column as a fucntion arguement and set other few columns as result of these functions?

Comment (carolinux) [inline reader.py]: Here is an example for your last point:
```
def apply_to_two_columns(df, func, cols=None, newcols=None):
    """ Apply a function to two columns (used mostly for lon/lat transformations) """
    if cols is None:
        col1 = "lon"
        col2 = "lat"
    else:
       col1 = cols[0]
       col2 = cols[1]

    if newcols is not None:
        new_col1 = newcols[0]
        new_col2 = newcols[1]
    else:
        new_col1 = col1
        new_col2 = col2

    df[new_col1], df[new_col2] = zip(*df.apply(lambda x:func(x[col1],x[col2]), axis=1))
    return df
```
